index.html

```diff
@@ -82,402 +83,384 @@
+          <li>
+            <div class="listing__card" data-reveal="bottom">
+              <div class="card-icon">
+                <img src="./assets/images/icon-5.png" width="71" height="71" loading="lazy" alt="icon" />
+              </div>
 
-                <div>
-                  <h3 class="headline-sm card-title">Anoskopia</h3>
+              <div>
+                <h3 class="headline-sm card-title">Anoskopia</h3>
 
-                  <p class="card-text">Loreme telerem</p>
-                </div>
+                <p class="card-text">Loreme telerem</p>
               </div>
-            </li>
+            </div>
+          </li>
 
-            <li>
-              <div class="listing-card" data-reveal="bottom">
-                <div class="card-icon">
-                  <img src="./assets/images/icon-6.png" width="71" height="71" loading="lazy" alt="icon" />
-                </div>
+          <li>
+            <div class="listing__card" data-reveal="bottom">
+              <div class="card-icon">
+                <img src="./assets/images/icon-6.png" width="71" height="71" loading="lazy" alt="icon" />
+              </div>
 
-                <div>
-                  <h3 class="headline-sm card-title">Gastroskopia</h3>
+              <div>
+                <h3 class="headline-sm card-title">Gastroskopia</h3>
 
-                  <p class="card-text">Possat weasgaw</p>
-                </div>
+                <p class="card-text">Possat weasgaw</p>
               </div>
-            </li>
+            </div>
+          </li>
 
-            <li>
-              <div class="listing-card" data-reveal="bottom">
-                <div class="card-icon">
-                  <img src="./assets/images/icon-7.png" width="71" height="71" loading="lazy" alt="icon" />
-                </div>
+          <li>
+            <div class="listing__card" data-reveal="bottom">
+              <div class="card-icon">
+                <img src="./assets/images/icon-7.png" width="71" height="71" loading="lazy" alt="icon" />
+              </div>
 
-                <div>
-                  <h3 class="headline-sm card-title">Wizyty domowe</h3>
+              <div>
+                <h3 class="headline-sm card-title">Wizyty domowe</h3>
 
-                  <p class="card-text">Lorem ipsum</p>
-                </div>
+                <p class="card-text">Lorem ipsum</p>
               </div>
-            </li>
-          </ul>
-        </div>
-      </section>
+            </div>
+          </li>
+        </ul>
+      </div>
+    </section>
 
-      <section class="section blog" aria-labelledby="blog-label">
-        <div class="container">
+    <section class="section blog" aria-labelledby="blog-label">
+      <div class="container">
 
 
-          <h2 class="section-title headline-md text-center" data-reveal="bottom">
-            Ostatnie wpisy na blogu
-          </h2>
+        <h2 class="blog__section-title headline-md text-center" data-reveal="bottom">
```

Comment:
Instead of using these helper classes, just write the declarations in css

```suggestion
        <h2 class="blog__section-title" data-reveal="bottom">
```